### PR TITLE
MGDCTRS-1850 chore: add a link to the SA guide

### PR DESCRIPTION
**Pull request by gashcrumb**
This commit adds some text and a link to the core configuration step to the user guide that covers service account creation and configuration for Kafka.  This change also adds reworks the core configuration step and adds some stories to storybook so it's easier to develop this step outside of the application.

Here's the storybook additions:
Screenshot from 2023-01-16 14-24-59

And when the user checks the checkbox:

Screenshot from 2023-01-16 14-26-41

Finally the review step change:
Screenshot from 2023-01-16 15-00-38

### CODE REVIEW

**Comment by gashcrumb** [comment]
@mmuzikar FYI this will probably break the tests.

File: cypress/e2e/connector-create.cy.ts
Comment (gashcrumb) [inline cypress/e2e/connector-create.cy.ts:209]: @mmuzikar here's one option

File: src/app/pages/CreateConnectorPage/StepCoreConfiguration.tsx
Comment (gashcrumb) [inline src/app/pages/CreateConnectorPage/StepCoreConfiguration.tsx:185]: @mmuzikar I added this as the other option for tests, just needs to be checked to continue through the wizard.

**Comment by mmuzikar** [comment]
@gashcrumb Thanks for keeping me in the loop. I'm thinking if it won't become too annoying to click the checkbox with every connector, maybe the approval could be stored in window storage and clicked only once?

**Review by indraraj — APPROVED** [review]
LGTM 👍

**Comment by indraraj** [comment]
Maybe we should also add this checkbox to the edit flow, and WDYT about making it default to true in the duplicate and edit flow? @gashcrumb

**Comment by gashcrumb** [comment]
I think not for the edit flow at this time, this is just a temporary band-aid.  Eventually we could maybe check the permissions or offer a button to do this manually, and that component would be the thing I'd want to use for the edit case.

However, it makes sense to me to have it checked for the duplicate flow, as I think of this checkbox as something to help a brand new user, and the duplicate flow is like a day 2 or day 3 thing that someone would do.  Will push an update for this.

**Comment by gashcrumb** [comment]
> @gashcrumb Thanks for keeping me in the loop. I'm thinking if it won't become too annoying to click the checkbox with every connector, maybe the approval could be stored in window storage and clicked only once?

This is a temporary solution.  Using local storage to store the value is good idea but at this stage I don't want to add too many parts to something that will likely be removed in the near future.

**Review by uidoyen — APPROVED** [review]
LGTM :+1:

**Comment by gashcrumb** [comment]
I'll merge this after #677 is merged as there will be some file conflicts.

**Pull request merged**
